stdlib/sys.pyi

```diff
@@ -278,7 +279,7 @@ if sys.platform == "win32":
 def intern(__string: str) -> str: ...
 def is_finalizing() -> bool: ...
 
-__breakpointhook__: Any  # contains the original value of breakpointhook
+__breakpointhook__ = breakpointhook  # Contains the original value of breakpointhook
```

Comment:
(Minor): can we move the definition of `__breakpointhook__` below the definition of `breakpointhook` here? The forward reference is a tiny bit confusing

Reply:
Sure!